src/handlers/deeplinks.ts

```diff
@@ -1,14 +1,14 @@
-import { captureException } from '@sentry/react-native';
+import qs from 'query-string';
```

Comment:
Cleaned up this file, easiest to look at the commit I did this in (one of the latest commits)

Reply:
❤️